docker/polygon_zkevm/polygon_zkevm.sh

```diff
@@ -8,9 +8,12 @@ ADDRESS=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
 PRIVKEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 
 CACHE_DIR=$(obtain_cache_dir $0)
+ABI_DIR="${ROOT}/contracts/.build"
+RPC_FILE="${ROOT}/deployments/config/local/rpc.json"
```

Comment:
Since `RPC_FILE` does not depend on the cache dir and is the same for all e2e test scripts, it would make sense to remove this variable, remove the corresponding argument to `deploy_beamer` function and instead make `deploy_beamer` from `common.sh` deteremine the RPC file location.